core/state/state_object.go

```diff
@@ -19,6 +19,7 @@ package state
 import (
 	"bytes"
 	"fmt"
+	substate "github.com/Fantom-foundation/Substate"
```

Comment:
nit: can we group std libraries and 3rd-party libraries separately? The 3rd-party libraries starting with `github.com/*` goes the second group (line 28-30).